### PR TITLE
misc. regular updates

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Explain config file types and locations

### CODE REVIEW

File: content/docs/command-reference/config.md [hunk 1]
```diff
@@ -19,32 +19,39 @@ You can query/set/replace/unset DVC configuration options with this command. It
...
+meant to be tracked by Git and should not contain sensitive and/or user-specific
+information (passwords, SSH keys, etc). Use the `--local` command option (flag)
+instead, to set (or override) secrets. `--global` and `--system` are also
+available to set config options for multiple projects and users, respectively.
+
+| Option     | Priority | Linux/Mac location     | Windows location                                          |
+| ---------- | -------- | ---------------------- | --------------------------------------------------------- |
+| `--local`  | 1        | `.dvc/config.local`    | `.dvc/config.local`                                       |
+| None       | 2        | `.dvc/config`          | `.dvc/config`                                             |
+| `--global` | 3        | `~/.config/dvc/config` | `%LocalAppData%/iterative/dvc/config`                     |
```
Comment (shcheklein) [inline content/docs/command-reference/config.md:37]: This is not correct for Mac (for global/system at least)

File: content/docs/command-reference/config.md [hunk 2]
```diff
@@ -19,32 +19,39 @@ You can query/set/replace/unset DVC configuration options with this command. It
...
+regular project's config file in `.dvc/config` is read or modified. This file is
+meant to be tracked by Git and should not contain sensitive and/or user-specific
+information (passwords, SSH keys, etc). Use the `--local` command option (flag)
+instead, to set (or override) secrets. `--global` and `--system` are also
+available to set config options for multiple projects and users, respectively.
+
+| Option     | Priority | Linux/Mac location     | Windows location                                          |
+| ---------- | -------- | ---------------------- | --------------------------------------------------------- |
+| `--local`  | 1        | `.dvc/config.local`    | `.dvc/config.local`                                       |
+| None       | 2        | `.dvc/config`          | `.dvc/config`                                             |
```
Comment (shcheklein) [inline content/docs/command-reference/config.md:36]: (default)

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/config.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/config.md:37]: Oh can you help me confirm what those are? I don't have a Mac with me rn, sry

**Comment by jorgeorpinel** [comment]
Huh? Weird. This PR is showing the changes of a previously merged one, https://github.com/iterative/dvc.org/pull/1783/files

File: content/docs/command-reference/config.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/config.md]: Removed "Mac" for now, merging this, but will update ASAP as I find out the right locations.

**Pull request merged**